jobs/config_test.py

```diff
@@ -373,6 +388,23 @@ def get_real_bootstrap_job(self, job):
         self.assertIn(key, sorted(self.realjobs))  # sorted for clearer error message
         return self.realjobs.get(key)
 
+    def test_non_blocking_jenkins(self):
+        """All PR non-blocking jenkins jobs are always_run: false"""
+        # ref https://github.com/kubernetes/test-infra/issues/4637
+        if not self.presubmits:
+            self.load_prow_yaml(self.prow_config)
+        required_jobs = get_required_jobs()
+        message_format = 'Jenkins jobs should not be `always_run: true`' + \
+            ' unless they are required! %s'
+        # TODO(bentheelder): should we also include othe repos?
+        # If we do, we need to check which ones have a deployment in get_required_jobs
+        # and ignore the ones without submit-queue deployments. This seems brittle
+        # and unnecessary for now though.
+        for job in self.presubmits.get('kubernetes/kubernetes', []):
+            if job['agent'] == 'jenkins' \
```

Comment:
I think we generally don't use `\`, instead you can do:
```
if ( foo and
     bar and
     baz )

Reply:
done. though pylint wants:
```
if (foo and
        bar and
        baz):
````
:confused:  :-1:

Reply:
Please line break after (, so this would be:

```
if (
        foo and
        bar and 
        baz):
```

This way if we change it to something like:
```
if somethingElse(
        foo and
        bar and 
        baz):
```
We don't have to change any of the other lines. Although admitedly that is less interesting in `if` cases than if it is `someFunctionThatMightChangeInTheFuture()`

Reply:
@fejta Makes sense --  I merged on LGTM but I can follow up with this or just leave it for next time ¯\\\_(ツ)\_/¯